csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -3476,46 +3484,148 @@ Example which fails the test:
+  * `/document/tracking/version`
+* The value of `/document/category` SHALL NOT be equal to any value that is intended to only be used by another profile. To explicitly select the use of this profile the value `generic_csaf` MAY be used.
+
+An issuing party might choose to set `/document/publisher/name` in front of a value that is intended to only be used by another profile to state that the CSAF document does not use the profile associated with this value. This should be done if the issuing party is able or unwilling to use the value `generic_csaf`, e.g. due to legal or cooperate identity reasons.
+
+> The value `Example Company Security Advisory` in `/document/category` uses the profile "Generic CSAF". This is important to prepare forward compatibility as later versions of CSAF might add new profiles. Therefore, the values which can be used for the profile "Generic CSAF" might change.
+
+## 5.2 Profile 2: Security incident response
+
+This profile SHOULD be used to provide a response to a security breach or incident. This MAY also be used to convey information about an incident that is unrelated to the issuing party's own products or infrastructure.
+
+> Example Company might use a CSAF document satisfying this profile to respond to a security incident at ACME Inc. and the implications on its own products and infrastructure.
+
+A CSAF document SHALL fulfill the following requirements to satisfy the profile "Security incident response":
+
+* The following elements must exist and be valid:
+  * all elements required by the profile "Generic CSAF".
+  * `/document/notes` with at least one item which has a `category` of `description`, `details`, `general` or `summary`
+    > Without at least one note item with information about response to the event referred to this doesn't provide any useful information.
+  * `/document/references`
+    > There should be a `reference` (link) to a document or website which provides more details about the incident.
+* The value of `/document/category` SHALL be `security_incident_response`.
+
+## 5.3 Profile 3: Informational Advisory
+
+This profile SHOULD be used to provide information which are **not related to a vulnerability** but e.g. a misconfiguration.
+
+A CSAF document SHALL fulfill the following requirements to satisfy the profile "Informational Advisory":
+
+* The following elements must exist and be valid:
+  * all elements required by the profile "Generic CSAF".
+  * `/document/notes` with at least one item which has a `category` of `description`, `details`, `general` or `summary`
+    > Without at least one note item with information about the "issue" which is the topic of the advisory it is useless.
+  * `/document/references`
+    > There should be a `reference` (link) to a document or website which provides more details about the issue or its remediation (if possible). This could be a link to a hardening guide, a manual, best practices or any other helpful information.
```

Comment:
As above, contradicts with `must`:
```suggestion
    > There MUST be a `reference` (link) to a document or website which provides more details about the issue or its remediation (if possible). This could be a link to a hardening guide, a manual, best practices or any other helpful information.
```

Reply:
We should not have formal language in an informative comment. Personally, I like the "MUST" part but I don't know how you want to test that automatically... So I would rather go with Should in an informative comment... It doesn't contradict the must from above as the presence of the value and its content is separated. 

@sthagen: Would do you think?

Reply:
I rephrased that.